### DISCUSSION TITLE
Feature Request: I would like to be able to **Tab** out of the Title textbox into the property inspector controls below

### DISCUSSION THREAD

**Discussion by MillerMark** [elgatosf/elgato#68] [Ideas]
Since the Stream Deck has come out, the **Title** text box has accepted the **Tab** key, which seems to make no sense at all since the Tab takes up so much white space, any text to the right of it cannot be seen in the button. So there's seemingly no point for an actual Tab to exist in the Title box.

Instead the Tab key should behave as universally expected - it should move focus to the next control in the tab order (down into the property editor controls). Similarly, Shift+Tab should move focus through the controls in reverse order.

Fixing this would make configuration for many buttons significantly faster for end users. Perhaps this fix might be something to consider for a future release.

**Comment by ChekTek** [comment]
Hi @MillerMark,

This is great feedback! I will share this with the Stream Deck team as a feature request.
